misc/iitthesis.cls

```diff
@@ -313,10 +343,6 @@
 \iitthesis@thesisdatafield{supervisionHebrew}{מידע על המנחים וההנחיה}
 \iitthesis@thesisdatafield{publicationInfoHebrew}{ציון אם העבודה פורסמה בכתבי עת או  הוצגה בכנסים. הרשימה תכתב בהתאם לכללי הציטוט (כולל הכותר ושמות השותפים). במקרה של שיתוף פעולה, פירוט תרומת המגיש.}
 \iitthesis@thesisdatafield{publicationInfoEnglish}{Info about whether the contents of the thesis was published, about collaborators and the thesis author's contribution.}
-\iitthesis@thesisdatafield{GregorianDateEnglish}{Gregorian Month and Year}
```

Comment:
You need to allow the user to override the year and month. So, these options - or some variation on them - must continue to exist.

* Q: Why?
* A: Because you might be typesetting the thesis _before_ the planned submission month, or _after_ it has already been submitted.